test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewResultTest.cs

```diff
@@ -97,6 +97,54 @@ public class ViewResultTest
             viewEngine.Verify();
         }
 
+        [Fact]
+        public async Task ExecuteResultAsync_ViewResultAllowNullViewDataAndTempData()
+        {
+            var context = GetActionContext();
```

Comment:
we usually decorate tests with `// Arrange`, `// Act` and `// Assert`. Have a look at other tests in this class.